assets/src/edit-story/components/richText/karma/inlineStyleOverride.karma.js

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { initHelpers } from './_utils';
+
+describe('CUJ: Editor Can Style Text', () => {
+  describe('Action: Inline style override', () => {
+    const data = {};
+
+    const {
+      getTextContent,
+      addInitialText,
+      setSelection,
+      richTextHasFocus,
+    } = initHelpers(data);
+
+    beforeEach(async () => {
+      data.fixture = new Fixture();
+      await data.fixture.render();
+
+      // Add a text box
+      await addInitialText();
+    });
+
+    afterEach(() => {
+      data.fixture.restore();
+    });
+
+    it('should have the correct initial text and formatting', () => {
+      expect(getTextContent()).toBe('Fill in some text');
+    });
+
+    describe('when cursor is placed after second character', () => {
+      beforeEach(async () => {
+        // Enter edit-mode
+        await data.fixture.events.keyboard.press('Enter');
```

Comment:
Maybe move entering to edit mode to `beforeEach` upper since that seems to be used in every test, below as well? Below it's done per test. Not a huge difference though.

Reply:
Moved in c9da0ae2f.